nifi-framework-api/src/main/java/org/apache/nifi/bootstrap/BootstrapCommunicator.java

```diff
@@ -25,11 +25,13 @@ public interface BootstrapCommunicator {
 
     /**
      * Sends a command with specific arguments to the bootstrap process
+     *
      * @param command the command to send
-     * @param args the args to send
+     * @param args    the args to send
+     * @return whether the command send was successful or not
      * @throws IOException exception in case of communication issue
      */
-    void sendCommand(String command, String... args) throws IOException;
+    boolean sendCommand(String command, String... args) throws IOException;
```

Comment:
For future flexibility, I recommend changing the `boolean` to an `enum` named something like `CommandResult` or `CommandStatus`. The initial values could be `FAILURE` and `SUCCESS`, but using an `enum` provides more options for extension with changing the interface contract.

Reply:
Fixed